spec/support/schemas/email_address_response.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
should I be able to see this in the "Models" section of the swagger docs?

Reply:
hmmm, good question.  @kreek @omgitsbillryan - Can you pls let me know what steps I would want to take to get the "Models" section of the swagger docs populated with the new email content?

Reply:
@hpjaj to get a model to show up in the swagger docs create a schema for it under `app/swagger/schemas` and then [$ref](https://github.com/department-of-veterans-affairs/vets-api/blob/1efd2c206859b1a261e762a50cdb44dc8b66462d/app/swagger/schemas/letters.rb#L17) it from the request.

Reply:
@kreek - Thanks!

Implemented in https://github.com/department-of-veterans-affairs/vets-api/pull/1718/commits/08ac137e566b2f097b524cbd9b759b8b3269cc17